### DISCUSSION TITLE
track event - event associated to an url only in "realtime" view

### DISCUSSION THREAD

**Discussion by boly38** [umami-software/umami#3009] [Q&A]
Hi,
using `umamiNodeSample.js` ([gist](https://gist.github.com/boly38/081318e620b6625467fef75bcb91b618#file-umaminodesample-js)) I produced a page hit + event hit for this page.

I am able them view it in umami cloud portal.
BUT the representation in "realtime" view (A) is not the same as in "event" view (2)

Realtime (A)
![image](https://github.com/user-attachments/assets/1c43d318-fd95-457f-baad-1c04e03ae056)

Events (B)
![image](https://github.com/user-attachments/assets/6ddbf76a-412f-413e-8b37-e708d1ce7ff7)

why from Events(B) view, we cant see the relation between event and page ?

Regards

**Comment by orendecor** [comment]
+1

I also think this is important to relation is also important in reports. For example when a I create funnel report I cannot choose only `button_click` events on page `x`
